src/tests/ftest/scripts/main.sh

```diff
@@ -186,7 +186,7 @@ if [ "${STAGE_NAME}" == "Functional Hardware 24" ]; then
 fi
 
 # shellcheck disable=SC2086,SC2090
-if ! ./launch.py --mode ci ${launch_node_args} ${LAUNCH_OPT_ARGS} ${TEST_TAG_ARR[*]}; then
+if ! ./launch.py --mode ci ${launch_node_args} ${LAUNCH_OPT_ARGS} "${TEST_TAG_ARR[*]}"; then
```

Comment:
You can't quote that.  I have fallen into that trap several times.

```suggestion
if ! ./launch.py --mode ci ${launch_node_args} ${LAUNCH_OPT_ARGS} ${TEST_TAG_ARR[*]}; then
```

Reply:
I'll just suppress it then as it's clearly working for now.

Reply:
I think there is a issue on the name of the variable:
s/TEST_TAG_ARR/TEST_TAG_ARG/

Reply:
I think TEST_TAG_ARG is a string of strings, TEST_TAG_ARR is an array of strings from TEST_TAG_ARG.